src/Terrain/terrain.cpp

```diff
@@ -473,25 +473,25 @@ float Terrain::get_H_cost(std::array<float, 3> xyz1, std::array<float, 3> xyz2)
     double D1 = 1.0;
     double D2 = 1.414;
     double D3 = 1.0;
-    auto [x1, y1, z1] = xyz1;
-    auto [x2, y2, z2] = xyz2;
+    //auto [x1, y1, z1] = xyz1;
+    //auto [x2, y2, z2] = xyz2;
```

Comment:
What's wrong with destructuring? Did you set your C++ version?

https://en.cppreference.com/w/cpp/language/structured_binding